src/utils/pythonSymbols.ts

```diff
@@ -26,69 +32,93 @@ export const getMainPythonUri = async () => {
     mainUri = pyFiles.length ? pyFiles[0] : undefined;
     if (mainUri) {
       workspaceConfig.update(TAIPY_STUDIO_SETTINGS_MAIN_PYTHON, workspace.asRelativePath(mainUri));
-      window.showInformationMessage(l10n.t("Main module file has been set up as {0} in Workspace settings", workspace.asRelativePath(mainUri)));
-      getLog().info(l10n.t("Main module file has been set up as {0} in Workspace settings", workspace.asRelativePath(mainUri)));
+      window.showInformationMessage(
+        l10n.t("Main module file has been set up as {0} in Workspace settings", workspace.asRelativePath(mainUri))
+      );
+      getLog().info(
+        l10n.t("Main module file has been set up as {0} in Workspace settings", workspace.asRelativePath(mainUri))
+      );
     } else {
       getLog().warn(l10n.t("No symbol detection as there is no Python file in workspace."));
     }
   }
   return mainUri || null;
 };
 
-export const getCreateFunctionOrClassLabel = (isFunction: boolean) => isFunction ? l10n.t("Create a new function") : l10n.t("Create a new class");
+export const getCreateFunctionOrClassLabel = (isFunction: boolean) =>
+  isFunction ? l10n.t("Create a new function") : l10n.t("Create a new class");
 
 export const MAIN_PYTHON_MODULE = "__main__";
 
-export const getModulesAndSymbols = async (isFunction: boolean): Promise<[string[], Record<string, string>]> => {
-  const workspaceConfig = workspace.getConfiguration(TAIPY_STUDIO_SETTINGS_CONFIG_NAME, workspace.workspaceFolders && workspace.workspaceFolders[0]);
+export const getModulesAndSymbols = async (
+  isFunction: boolean
+): Promise<[string[], Record<string, string>, string | undefined]> => {
```

Comment:
Not too sure why suddenly this would return an undefined value, compared to before, but hey.

Reply:
if there's no main module specified the main module path will be undefined